src/api/java/com/direwolf20/buildinggadgets/api/abstraction/IApiConfig.java

```diff
@@ -0,0 +1,4 @@
+package com.direwolf20.buildinggadgets.api.abstraction;
+
+public interface IApiConfig {
```

Comment:
No code?

Reply:
So far there isn't any implementation, which would require configs but future might find some...
If you want to we can exclude that easily for now...

Reply:
It's fine, can you just add a comment explaining the use of the interface and why it's empty at this point using a @since